package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "@babel/plugin-proposal-decorators": "7.7.4",
     "@react-native-community/async-storage": "1.7.1",
+    "@react-native-community/masked-view": "0.1.6",
```

Comment:
why wee need masked view?

Reply:
After upgrading `react-navigation` and `react-navigation-stack` I started getting errors about this dependency.
Similar case to this one: https://github.com/react-navigation/stack/issues/323